scripts/test/index.js

```diff
@@ -0,0 +1,7 @@
+const moment = require('moment');
+
+const date = moment().add('5 minutes').toDate();
+console.log(date);
+
+const date2 = moment().add('5', 'm').toDate();
+console.log(date2);
```

Comment:
?

Reply:
This is why you have reivews....
I've comitted a script that I was using for testing.

Reply:
nah, thats why you self-review before requesting ;)

Reply:
Not if you are in a hurry and don't do it properly 😄 

Therefore we have a second layer of security.